### PR TITLE
fixed bug that was missing some engineering roles, refactored to find…

### PR DESCRIPTION
**Pull request by CodingHobo**
fixed bug that was missing some engineering roles, refactored to find parent div where job link lived

### CODE REVIEW

**Review by MGHermanMancarella — APPROVED** [review]
Look'n good!

File: ashby_scraper.py [hunk 1]
```diff
@@ -47,11 +47,12 @@ def scrape_ashby_job_board(url):
...
 
     # ashby usually has job listings within <div> elements with a class of "posting"
-    for job_div in soup.find_all('div', class_='ashby-job-posting-brief-list'):
+    for job_div in soup.find_all('div', class_='ashby-job-posting-brief'):
         job_title = job_div.find('h3').get_text()
         location = job_div.find('p').get_text().split("•")[1].strip()
-        job_url = BASE_URL + job_div.find('a')['href']
+        job_url = BASE_URL + job_div.parent['href']
         job_id = job_url.split("/")[-1]
+        print("JOB TITLE ===> ", job_title)
```
Comment (MGHermanMancarella) [inline ashby_scraper.py:55]: can you remove this print statement, plz? tyty

Comment (CodingHobo) [reply ashby_scraper.py]: Done

**Review by MGHermanMancarella — COMMENTED** [review]
approval round 2

**Pull request merged**